### ISSUE TITLE
With "autostart scanner", cannot reach home screen

### ISSUE THREAD

**Issue by dingens** [Fr4gorSoftware/SecScanQR#220]
When the "autostart scanner" option is on, it is impossible to reach the home screen and the other tasks.

**To reproduce:** with that option on, launch the app.

**Actual results:** Pressing backspace or the back arrow, I end up at a useless "you canceled the scanning" page with only two buttons, "back" and "scan", which both take me to the scanning screen again.

**Expected results (quick fix):** take me to the home screen immediately when pressing back / back arrow while in scan mode. (That is, remove that "you canceled" page -- or is there a reason for its existence i oversaw?)

**Expected results (premium solution)** (I can make a separate issue on this with a mockup, if desired)
There's only four (other) buttons in the main screen. So actually the main screen is not necessary IMHO. Just include those four buttons on the scan screen and make the scan screen the home screen (it's the mostly used thing anyway, I'd say), and remove that option.

**Comment by bigConifer** [comment]
Good point. I'd also suggest to remove this "start scan" screen completely.
However this issue is titled a bit inappropriate (you CAN, by simply pressing backspace once again).
[Here](https://github.com/Fr4gorSoftware/SecScanQR/issues/222) I've opened a new issue with more detailed description.

**Comment by georg-d** [comment]
I can't reproduce the actual results – when pressing back button in scanning screen, I get a short help, pressing back button again brings me to the main menu while pressing arrow left brings me back to scanning. IMHO that's OK. Saying that, I'd clearly favor the "premium solution" – BTW nice wording 🙂 – because it is more clear and requires less clicks.
